encrypted-media-respec.html

```diff
@@ -832,44 +832,65 @@
                   </li>
                   <li><p>Let <var>parameters</var> be the RFC 6381 [[!RFC6381]] parameters, if any, specified by <var>content type</var>.</p></li>
                   <li><p>If the user agent does not recognize one or more <var>parameters</var>, continue to the next iteration.</p></li>
-                  <li><p>Let <var>media types</var> be the set of codecs, with parameters, specified by <var>parameters</var>. It MAY be empty. The case-sensitivity of string comparisons is determined by the appropriate RFC or other specification.</p>
+                  <li><p>Let <var>media types</var> be the set of codecs and codec constraints specified by <var>parameters</var>. The case-sensitivity of string comparisons is determined by the appropriate RFC or other specification.</p>
                     <p class="note">Case-sensitive string comparison is RECOMMENDED because RFC 6381 [[RFC6381]] says, "Values are case sensitive" for some formats.</p>
                   </li>
+                  <li>
+                    <p>
+                      If <var>media types</var> is empty:
+                      <dl class="switch">
+                        <dt>If <var>container</var> implies a specific set of codecs and codec constraints:</dt>
```

Comment:
"normatively implies" ? At least that suggests that it needs to be a concrete specified thing, not just a common practice